docs/guide/contributing.rst

```diff
@@ -1,73 +1,203 @@
+Follow these guidelines for discussing tools: 
+
+* Ensure your content is of interest to a general audience.
+* Avoid advocacy and plugs about your favorite toolchain, even if it's open source.
+* Present specific use cases, how problems were solved, and what worked or didn't work well. 
+* Consider to first `file a GitHub issue`_ or contact one of the guide editors at guide@writethedocs.org.
+
+Keep content focused on general principles and best practices instead of arguing over minor 
+points at the expense of major principles of clarity and communication. Avoid preferential 
+editing that only offers your personal preferences for dealing with a particular writing 
+situation. For example, if you recommend a particular word choice, tell the audience why it 
+matters.
+
+Use a friendly and encouraging tone. It's a good way to write docs in general, so practice 
+with docs about docs!
+
+When linking to resources, attribute the links appropriately and mention why the resources 
+are useful.
+
+Some files in the guide are written in `Markdown`_, and some 
+are written in `reStructured Text (rST)`. We prefer rST 
+although either format is acceptable.
+
+Prerequisites
+--------------
+
+1. `Create a GitHub account`_.
+2. `Download and install Git`_.
+3. Open a terminal window and follow the instructions to `associate your
+   GitHub username with your local Git installation`_.
+
+   1. In macOS: Open the **Terminal** app.
+   2. In Windows: From the Start Menu, open **Git Bash**.
+
+4. In the repo UI, find the file name corresponding to an `existing issue`_ or page you want to improve. 
+
+For example, ``/docs/documentarians.rst`` creates https://www.writethedocs.org/documentarians/. 
+
+5. Review formatting guidelines for the Markup style the file uses. The example file above uses `reStructuredText (.rst) Markup`_. Other Markup styles for our guides include Markdown (.md).
+
+.. _update-guide-in-git:
+
+Update a guide in Write the Docs via a pull request
+----------------------------------------------------
+
+If you've never worked with Git and GitHub, use this section to get started. 
+
+**In the GitHub UI:**
+
+1.  `Fork the Write the Docs www project`_.
+
+2.  Click **Create fork**.
 
-* More :ref:`api-documentation` content!
-* Helping non-writers understand what makes good documentation
-* Helping writers new to developer tools or docs-as-code workflows to understand these toolchains and workflows
-* High-level discussions of common tools and toolchains. Please avoid advocacy here! But thoughtful presentation of specific use cases, how you solved them, what worked well, and what didn't work so well, are most welcome. If you want to write this kind of topic, consider starting with an issue in GitHub, or contact one of the new guide editors.
+3.  Click **< > Code**. 
 
-If you didn't find a topic you were looking for, file an `issue in GitHub <https://github.com/writethedocs/www/issues>`_.
+4.  Copy the HTTPS URL for cloning the repository.
```

Comment:
But this doesn't actually do the thing, so wont help the folks who we're addressing